<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autómata Celular v0.6.3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" rowspan="8" colspan="9">
     <widget class="QTableWidget" name="Data_Table">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>140</number>
      </attribute>
      <column>
       <property name="text">
        <string>NOMBRE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CANTIDAD INICIAL</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DIRECT FITNESS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>INDIRECT FITNESS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ASOCIACIÓN</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>AGRUPACIÓN</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>FLEXIBILIDAD</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="6" column="10">
     <widget class="QSpinBox" name="Reproduction">
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="Continue_Button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Continuar simulación</string>
      </property>
     </widget>
    </item>
    <item row="11" column="7">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="10">
     <widget class="QSpinBox" name="Nodes">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="11" column="6">
     <widget class="QLineEdit" name="CSV_NAME"/>
    </item>
    <item row="11" column="3">
     <widget class="QPushButton" name="Load_Button">
      <property name="text">
       <string>Cargar Configuración</string>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QPushButton" name="Save_Button">
      <property name="text">
       <string>Guardar Configuración</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="11">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="10">
     <widget class="QPushButton" name="EXIT_Button">
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </item>
    <item row="3" column="10">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>RECURSOS POR CICLO</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="5">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>NOMBRE DEL CSV</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="Start_Button">
      <property name="text">
       <string>Iniciar simulación</string>
      </property>
     </widget>
    </item>
    <item row="4" column="10">
     <widget class="QSpinBox" name="Resources">
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="7" column="10">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>MUERTES POR SELECCIÓN DE GRUPO (%)</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="8">
     <widget class="QPushButton" name="Reset_Button">
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9" rowspan="9">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>CANTIDAD DE NICHOS</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>NICHOS</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="9">
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ESPECIES</string>
      </property>
     </widget>
    </item>
    <item row="5" column="10">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>CONSUMO EN FUNCIÓN DEL INCLUSIVE FITNESS (%)</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="10">
     <widget class="QDoubleSpinBox" name="Deaths">
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
